src/dev/ci_setup/setup.sh

```diff
@@ -180,3 +180,28 @@ if [ "$GIT_CHANGES" ]; then
   echo -e "$GIT_CHANGES\n"
   exit 1
 fi
+
+###
+### github-checks-reporter kill switch. Remove to disable
+###
+export CHECKS_REPORTER_ACTIVE=true
+
+### only run on pr jobs
+if [[ "$JOB_NAME" != "elastic+kibana+pull-request"* ]] ; then
+  export CHECKS_REPORTER_ACTIVE=false
+fi
+
+###
+### Implements github-checks-reporter kill switch when scripts are called from the command line
+### $@ - all arguments
+###
+function checks-reporter-with-killswitch() {
+  if [ "$CHECKS_REPORTER_ACTIVE" = true ] ; then
```

Comment:
Lets use `"$CHECKS_REPORTER_ACTIVE" == "true"` instead, I'm kinda shocked this works and it totally looks like it will do assignment, also idk how bash handles type conversion.